migrate.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
```

Comment:
As a note for later, I want to make a proper CLI for the system (especially if @pyslackers/sir-bot-a-lot comes into the fold). We could do things like code generation (plugins, endpoints, etc)